fiftyone/core/threed/lights.py

```diff
@@ -68,6 +69,7 @@ class AmbientLight(Light):
     This light globally illuminates all objects in the scene equally.
 
     Args:
+        name ("AmbientLight"): the name of the light
```

Comment:
since we are setting the default value to the class name in the base `Light` class, we do not need to explicitly set this `name` value for any child class anymore :)

Reply:
We don't, but omitting this means `name` arg isn't explicit in the docstring / API. This way you don't have to hop back in the inheritance tree (eg: `PointLight -> Light -> Object3D`). We do this for other attributes like `position`, `quaternion`, etc.